ccd-definition/CaseEvent.json

```diff
@@ -241,6 +263,9 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "Submitted",
     "PostConditionState": "Submitted",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
```

Comment:
What about adding `"ShowSummary": "N", "ShowEventNotes": "N",` explicitly to every event?

The fact that event notes are hidden for users in solicitor role is rather surprising to me.